src/plugins/data/public/ui/query_editor/query_editor_top_row.tsx

```diff
@@ -225,7 +225,17 @@ export default function QueryEditorTopRow(props: QueryEditorTopRowProps) {
   }
 
   function shouldRenderDatePicker(): boolean {
-    return Boolean(props.showDatePicker ?? true) ?? (props.showAutoRefreshOnly && true);
```

Comment:
whoops we lost this `showAutoRefreshOnly`

Reply:
Discussed offline, was originally fixing a regression since this conditional does not reach `(props.showAutoRefreshOnly && true)` as `Boolean(props.showDatePicker ?? true)` would never be undefined. Fixed the regression by adding it back in as discussed with @kavilla in [1640375](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8866/commits/1640375843f5ff28537f9461969b7cb11d14d3d6).